ui/widgets/sidebar/src/components/sidebar-component.tsx

```diff
@@ -55,12 +55,12 @@ const SidebarComponent: React.FC<unknown> = () => {
   const routing = plugins['@akashaorg/app-routing']?.routing;
 
   const profileName = useMemo(
-    () => (!isLoggedIn || isLoading ? t('Guest') : myProfileQuery.data?.akashaProfile?.name || ''),
-    [myProfileQuery.data, isLoggedIn, isLoading, t],
+    () => (!isLoggedIn ? t('Guest') : myProfileQuery.data?.akashaProfile?.name || ''),
+    [myProfileQuery.data, isLoggedIn, t],
   );
 
   //empty profile name implies the connection process is still in progress
-  const inProgress = isLoading || !profileName;
+  const inProgress = isLoading || (isLoggedIn && myProfileQuery.status === 'loading');
```

Comment:
This line requires revision. You can refer to the comment I shared here https://github.com/AKASHAorg/akasha-core/pull/2079#issuecomment-1774711653.

Reply:
I understood the change you made :) But I was trying to bring to your attention an issue I observed which I shared on the comment. Here is a screen recording of it.

https://github.com/AKASHAorg/akasha-core/assets/35398733/d44d6561-0b7e-4caa-bd8d-bba751e268e7

After you login and refresh the page there is a gap before the logged in profile avatar shows and we aren't showing a loading placeholder. We need to revise the logic for checking `inProgress`.